### DISCUSSION TITLE
Add option Read aloud feature and Double click or selected text to translate

### DISCUSSION THREAD

**Discussion by huuson889** [immersive-translate/immersive-translate#2254] [Ideas]
### 新功能 | New Feature

1. Read aloud feature
2. Double click or selected text to translate
![{91ABBD57-4057-4727-B6E0-2E4F30C3A23D}](https://github.com/user-attachments/assets/937d5d80-7311-4535-aa3e-dd62c8ea3075)

### 为什么您想要添加这些功能 | Why you want to add these features

1. Read aloud help people know how to pronounce the words
2. Some time, we need to know exactly one or some word meaning. Selected feature will help us do that
3.

### 提议 | Proposal

![{C77F21D2-BE60-41F2-9555-158E3B0DA9E8}](https://github.com/user-attachments/assets/e6a2c629-5422-425a-ae55-b9bb04c07c11)

**Comment by theowenyoung** [comment]
Sorry, the word segmentation translation feature is currently not on the roadmap for immersive translate. You can use other plugins to replace this function.
